apps/shooting-clubs/views/location-address-category.html

```diff
@@ -0,0 +1,9 @@
+{{<partials-page}}
+  {{$page-content}}
+    {{#fields}}
+      {{#renderField}}{{/renderField}}
+    {{/fields}}
+    {{#input-submit}}continue{{/input-submit}}
+    {{> partials-add-another-address-summary}}
+  {{/page-content}}
+{{/partials-page}}
```

Comment:
This looks *very* similar to https://github.com/UKHomeOffice/firearms/blob/master/apps/common/views/add-another-address-loop.html - it's probably better to re-use that than create a new template.